.github/workflows/on-demand-build-docker-image-deploy-preview.yml

```
@@ -104,8 +104,7 @@ jobs:
                 "🚨 *Note*: The release environment runs EE code, so using a frontend-only DP on this repo, will",
                 "likely behave unexpectedly. Consider using a full DP instead.",
                 "[Learn more](https://notion.so/031b87bce3404e3a95240d4c14c82e46).",
-              ]
-            }
+            )}
```

Formatting gets messed up with this. The brace and paren don't open and the same level, or on the same line. Let's please keep them aligned, or the code gets quite confusing to read.

```suggestion
              )
            }
```